lib/inets/src/http_server/httpd_conf.erl

```diff
@@ -75,12 +75,23 @@ load_mime_types(MimeTypesFile) ->
 	    {error, ?NICE("Can't open " ++ MimeTypesFile)}
     end.
 
+%% Add any optional properties if they are not present.
+optional_properties(Properties) ->
+    case proplists:get_value(server_name, Properties) of
+        undefined ->
+            {ok, Fqdn} = net_adm:dns_hostname(net_adm:localhost()),
```

Comment:
I believe net_adm:localhost()  would be a better default as it would work for nodes with short names. 

Despription of  net_adm:localhost(): 
Returns the name of the local host. If Erlang was started with command-line flag -name, Name is the fully qualified name.

Reply:
Thanks for the review.

Revisiting this I actually was not quite sure why I wrapped it into `dns_hostname`. I have amended the commit to only use `net_adm:localhost()` on its own.